tmp/build/patch-jaeger.sh

```diff
@@ -5,6 +5,9 @@ function jaeger_patch_values() {
   sed -i -e '/service:$/,/externalPort:/ {
     s/name:.*$/name: zipkin/
 	}' -e '/jaeger:$/,/^[^ 	]/ {
+          /jaeger:/a\
+  # include elasticsearch to support default configurations\
+  elasticsearch: {}
```

Comment:
Indentation is looking funny here, looking into that right now

Reply:
I think there is a mixture of tabs and spaces within that file, I didn't tidy that up as part of this commit.